### ISSUE TITLE
Fix `event restart` command

### ISSUE THREAD

**Issue by kdhageman** [aau-network-security/haaukins#156]
Unable to restart resources of a team on an event, 
` hkn event restart <event-tag> <team-id>` command does not work as intended !

**Comment by mrturkmen06** [comment]
I think, there is a strong relationship with the issue #210. Fixing memory leaks caused by unstopped VMs when event is closed will help us to easily fix this issue as well.  Trying to make test statements on Stop() function...

**Comment by mrturkmen06** [comment]
The bug has been solved on branch [restart-team-functionality-fix](https://github.com/aau-network-security/haaukins/tree/restart-team-functionaly-fix) and created pull request into branch [white-scree-minor-fix](https://github.com/aau-network-security/haaukins/tree/white-scree-minor-fix), at the end these branches will be merged into [1.6.0](https://github.com/aau-network-security/haaukins/tree/1.6.0)

**Comment by mrturkmen06** [comment]
Reopened, after having refactored lab.hub this bug re-appeared, whenever command `hkn event restart <event-tag> <team-id>` used it returns `Could not find lab by the specified tag` response.

**Comment by mrturkmen06** [comment]
A dirty fix has been implemented in branch [dirty-fix-restart-team-lab-#156](https://github.com/aau-network-security/haaukins/tree/dirty-fix-restart-team-lab-%23156), it restarts resources of team only once, when ` hkn event restart <event-tag> <team-id> ` command tried for the second time, daemon is complaining about DNSCore container by saying `container is not running...`.

**Comment by mrturkmen06** [comment]
fixed on commit c66e086
